### PR TITLE
Add live active mic icon

### LINKED ISSUE #102: Active Microphone Icon
Add an animatable active microphone icon.

Design spec to come.

### PR DESCRIPTION
**Pull request by philschanely**
Resolves #102 

NOTE: This implementation shows the functionality on "click" but the actual implementation int he Live app will work together with live audio data. 

- [x] Mock up animating microphone icon
- [x] Sync with design

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]
👍 Nice work! It looks to be working as expected.

File: app/assets/stylesheets/sage/patterns/objects/_live_active_mic.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/* ==================================================
+  ** live_active_mic
+  type: object
+
+
+================================================== */
+
+.sage-live-active-mic {
+  --color-volume: #{sage-color(sage)};
```
Comment (teenwolfblitzer) [inline app/assets/stylesheets/sage/patterns/objects/_live_active_mic.scss]: Should there be a fallback value for the custom props with IE11? The image does display as a static icon just fine (no animation) in IE11, so it's probably reasonable to treat the animation as an "enhancement".

**Review by teenwolfblitzer — APPROVED** [review]
👍 😎 Looks great! Left one question about IE11 support that may be optional.

File: app/assets/stylesheets/sage/patterns/objects/_live_active_mic.scss (hunk 1 above)
Comment (philschanely) [reply app/assets/stylesheets/sage/patterns/objects/_live_active_mic.scss]: As "Sexy" as this approach is perhaps I should just switch to using direct properties. 🤔

**Review by AndrwM — APPROVED** [review]
This is nice! +1

**Comment by philschanely** [comment]
Y'all, I decided to back out from a good bit of the custom prop stuff, leaving only the volume value. And that itself is simpler now as well to make for animating the value much simpler in an IE-11 compatible environment. Docs are updated too. Lets land this!

**Pull request merged**